### DISCUSSION TITLE
OperatorHub displays only the latest version (0.23)

### DISCUSSION THREAD

**Discussion by moondly** [strimzi/strimzi-kafka-operator#5209] [Q&A]
I noticed a user cannot select anymore an older version on Strimzi when using the operator hub. The only possibility is the latest version (0.23). Before it was possible to select older versions like "0.19.."

Is this a bug or a hidden assumption for this new version?

Thanks

<img width="286" alt="Screenshot 2021-06-28 at 19 29 01" src="https://user-images.githubusercontent.com/49954124/123678870-2762dc00-d847-11eb-8621-7854f0cf876d.png">

**Comment by scholzj** [comment]
AFAIK that was always like that. When you click install, you should be able to select different channels with different versions. But the Operator hub IMHO never allowed you to select version directly. And if it did and I didn't noticed it, I think this would be an OpenShift change, not anything we did.

**Reply by moondly** [reply]
I'm pretty sure it was possible to select a channel in this page. I did this multiple times in the past. But, if it is not related to any change in Strimzi, it may be something with Openshift. I will have to check other possibilities then. Tks.

**Reply by scholzj** [reply]
Hmm, I know you can do that on https://operatorhub.io/operator/strimzi-kafka-operator ... I do not remember seeing this on OpenShift. In any case, I'm not aware of any change form our side which would cause this. So it might be an OpenShift change.

**Reply by moondly** [reply]
My mistake here. Something was inconsistent on my env. I see the correct flow now. Thanks for the support.
